src/hooks.client.ts

```diff
@@ -18,7 +18,12 @@ Sentry.init({
 	replaysOnErrorSampleRate: 1.0,
 
 	// If you don't want to use Session Replay, just remove the line below:
-	integrations: [new Replay()],
+	integrations: [
+		new Sentry.BrowserTracing(),
```

Comment:
The Sveltekit SDK already adds `BrowserTracing` (not `BrowserProfiling` though) to the integrations. Did this not work correctly for you?

Reply:
Gotcha, so if using Sentry.init from `'@sentry/sveltekit'` we don't need to add the integration? TBH I just saw that I wanted to add BrowserTracing, went to the docs, saw the instructions and added.

Reply:
> so if using Sentry.init from '@sentry/sveltekit' we don't need to add the integration?

correct you only need to add `BrowserProfiling`.

Ahh I just saw that the [browser profiling docs](https://docs.sentry.io/platforms/javascript/guides/sveltekit/profiling/#configure) include `BrowserTracing`. Thanks for letting me know, I'll fix this in the docs.

Reply:
Glad to help. Thanks for the info.